packages/types-database/src/types/client.ts

```diff
@@ -43,8 +43,74 @@ export const UserDataSchema: mongoose.Schema<UserDataRecord> = new Schema({
 	updatedAtTimestamp: Number,
 });
 
+type User = {
```

Comment:
do these need to be in here? wrt types, everything should be in private by default unless needed by the provider imo

Reply:
I could remove the client getter job and then these wouldn't be needed. We'd be relying on the API `registerSiteKey` stuff always working in this case. Shall I remove?